packages/e2e/features/ui/components/storage/storage-browser/action-menu.feature

```diff
@@ -135,7 +133,4 @@ Feature: Create folder with Storage Browser
     Then I click the "Delete" menuitem
     Then I click the "Delete" button
     Then I see "All files deleted"
-    When I click the "Exit" button
-    # verify all files are deleted
-    Then I see "No files"
```

Comment:
Info: is there a test for deletion where this logic exists? It seems like it would make sense to have a confirmation that deleted files are then not listed

Reply:
We're testing delete on [upload, list, copy and delete a file using toggle menu](https://github.com/aws-amplify/amplify-ui/blob/main/packages/e2e/features/ui/components/storage/storage-browser/action-menu.feature#L66C13-L66C67) test. It has a `Then I see "All files deleted"` step to verify the deletion